tests/benchmarks/micro_benchmarks/test_memory_bw_latency_performance.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Tests for kernel-launch benchmark."""
+
+import numbers
+
+from tests.helper import decorator
+from superbench.benchmarks import BenchmarkRegistry, BenchmarkType, ReturnCode
+
+
+@decorator.cuda_test
```

Comment:
what should be the correct value for this?

Reply:
I think it's not a benchmark relying on GPU? Please just remove this line.

Reply:
Done